source/slang/slang-emit.cpp

```diff
@@ -664,18 +664,7 @@ Result linkAndOptimizeIR(
         break;
     }
 
-    switch (target)
-    {
-    case CodeGenTarget::CPPSource:
-    case CodeGenTarget::CUDASource:
-        break;
-    default:
-        // For all targets that don't support true dynamic dispatch through
-        // witness tables, we need to eliminate witness tables from the IR so
-        // that they don't keep symbols live that we don't actually need.
-        stripWitnessTables(irModule);
-        break;
-    }
+    stripWitnessTables(irModule);
```

Comment:
Maybe a `TODO` or `Note` comment is warranted, to point out that we currently always strip witness tables because our dynamic dispatch no longer needs them, but if/when we have targets that support dynamic dispatch with separately-compiled functions then we might need to leave them in place?